frontend/app/src/index.tsx

```diff
@@ -19,13 +19,17 @@ const streamlitExecutionStartedAt = Date.now()
 
 import React from "react"
 
+import log from "loglevel"
 import ReactDOM from "react-dom"
 import { Client as Styletron } from "styletron-engine-atomic"
 import { Provider as StyletronProvider } from "styletron-react"
 
 import ThemedApp from "./ThemedApp"
 
 const engine = new Styletron({ prefix: "st-" })
+if (process.env.NODE_ENV === "development") {
+  log.setLevel(log.levels.DEBUG)
+}
```

Comment:
Whats the default log level for a normal app?

Reply:
Logging is filtered to "warn" level by default. Will add a comment.